pkg/genericapiserver/OWNERS

```diff
@@ -1,4 +1,41 @@
-assignees:
-  - lavalamp
-  - nikhiljindal
-  - smarterclayton
+approvers:
+- lavalamp
+- nikhiljindal
```

Comment:
Add deads2k to this list - he's effectively the owner and is leading the Sig on this.  Otherwise lgtm